README.Rmd

```diff
@@ -48,7 +48,7 @@ To initialize the branding config, use `brand_init()`:
 brand_init()
 ```
 
-This will generate the `config.yml` and `_brand.yml` files. The `_brand.yml` file contains placeholder text. You will need to edit the config file with the URL of your brand file, then download that file using:
+This will generate the `rbranding_config.yml` and `_brand.yml` files. The `_brand.yml` file contains placeholder text. You will need to edit the config file with the URL of your brand file, then download that file using:
```

Comment:
I wonder, could we use the `_brand.yml` file to add the information we need for the `rbranding_config.yml` file. I guess that the answer should be "yes". I believe the brand.yml standard allows passing additional metadata.

Reply:
Also, something for later (hopefully before the CRAN release!).

Reply:
As an idea, you could be pulling `get_brand()` without having the url that originated the file, but then add the url to the local copy. When comparing the file, you could simply remove the extra metadata and then perform the hash comparison.